### DISCUSSION TITLE
Nordigen import fails with old configuration file

### DISCUSSION THREAD

**Discussion by Zap0** [firefly-iii/firefly-iii#7241] [General]
Was troubleshooting my data import not working, and it turned out to be the old config file (October last year) I was using for the import. The error message I was getting was nothing but "look at logs" (I don't even know where they are). The import process failed before even redirecting me to the respective third party auth pages. Fortunately it was easy enough to fix by just not using the old config file.

Perhaps there could be versioning or validation of the config files? I've not looked into what actually was the problem with the config I was using, but it was still working last month using old firefly and importer versions from October.

**Comment by JC5** [comment]
All config files should be backwards compatible. Keep in mind that Nordigen requisition ID's expire. 

Send me your config file and I can validate my own statement 😅

**Reply by Zap0** [reply]
`accounts` and `nordigen_requisitions` are the only entries different between my configs, yeah. Do we know when the requisition IDs expire and can give a message to that effect perhaps?

**Reply by JC5** [reply]
Yes, and it should. There's a specific ExpiredException. It depends on the version of your data importer if it will show you this as a message or an error. Either way, you can grab the logs like so:

https://docs.firefly-iii.org/data-importer/faq/#how-can-i-debug-the-data-importer
